fastlane/Fastfile

```diff
@@ -20,7 +20,7 @@ platform :android do
   lane :test do |options|
     flavor=options[:flavor].capitalize
 	type=options[:type].capitalize
-    gradle(task: "test")
+    gradle(task: "test#{flavor}#{type}UnitTest")
```

Comment:
Seems like there was an error and the flavor/type weren't actually specified for the tests, which resulted in all tests for all flavors and types being run every time.

I think my original idea was to only run flavor/type that was requested for the build itself. This should reduce the amount of time necessary to make a manual distribution build.